core-api/src/main/java/com/sequenceiq/cloudbreak/doc/ModelDescriptions.java

```diff
@@ -241,6 +241,7 @@ private ModelDescriptions() {
         public static final String TUNNEL = "Configuration that the connection going directly or with cluster proxy or with ccm and cluster proxy.";
         public static final String FLOW_ID = "Flow identifier for the current stack creation. Only returned during the stack create request/response.";
         public static final String EXTERNAL_DATABASE = "External database parameters for the stack.";
+        public static final String SDX_CLUSTER_SHAPE = "SDX Cluster Shape";
```

Comment:
This is a very basic description. Can you add more information, like explaining what a cluster shape is?